webapp/src/views/projects/translations/Namespace/NamespaceBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useRef } from 'react';
+import { styled } from '@mui/material';
+import { useHeaderNsDispatch } from '../context/HeaderNsContext';
+import { NamespaceContent } from './NamespaceContent';
+import { NsBannerRecord } from '../context/useNsBanners';
+
+const StyledNsRow = styled('div')`
+  display: flex;
+  background: ${({ theme }) => theme.palette.emphasis[50]};
+  height: 17px;
```

Comment:
According the design, it should have height 1px. Last in ns and last in ns should be identified and should have larger padding-bottom/top to create space for the banner.